groups/reconcile.go

```diff
@@ -131,6 +131,9 @@ func main() {
 	confirmChanges := flag.Bool("confirm", false, "false by default means that we do not push anything to google groups")
 	printConfig := flag.Bool("print", false, "print the existing group information")
 	numWorkers := flag.Int("workers", defaultNumWorkers, "number of concurrent workers to use")
+	sourceGroupEmail := flag.String("sourceGroup", "", "Email address of the source google group")
+	destinationGroupEmail := flag.String("destinationGroup", "", "Email address of the destination google group")
```

Comment:
Non-blocking nit
I'd prefer flag names to be as `source-group` and `destination-group`.

Reply:
updated to ^ suggested flags. Thanks! :slightly_smiling_face: